sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -85,19 +85,7 @@ internal static void AddPropertiesToTelemetry(IDictionary<string, string> destin
                 for (int i = 0; i < UnMappedTags.Length; i++)
                 {
                     var tag = UnMappedTags[i];
-                    if (tag.Key.Length <= SchemaConstants.KVP_MaxKeyLength && tag.Value != null)
-                    {
-                        // Note: if Key exceeds MaxLength or if Value is null, the entire KVP will be dropped.
-                        // In case of duplicate keys, only the first occurence will be exported.
-#if NET6_0_OR_GREATER
-                        destination.TryAdd(tag.Key, Convert.ToString(tag.Value, CultureInfo.InvariantCulture).Truncate(SchemaConstants.KVP_MaxValueLength) ?? "null");
-#else
-                    if (!destination.ContainsKey(tag.Key))
-                    {
-                        destination.Add(tag.Key, Convert.ToString(tag.Value, CultureInfo.InvariantCulture).Truncate(SchemaConstants.KVP_MaxValueLength) ?? "null");
-                    }
-#endif
-                    }
+                    TryAddKvp(destination, tag);
```

Comment:
This method name is very generic, name it according to its operation.

Reply:
renamed to `AddKvpToDictionary`